dev/release/rat_exclude_files.txt

```diff
@@ -10,6 +10,8 @@ ci/linux-packages/changelog
 ci/linux-packages/*.install
 csharp/*.sln
 csharp/*.csproj
+csharp/*.md
+csharp/*.sql
```

Comment:
Can we add the license header via a markdown/sql comment?

Reply:
yes, I think that could work.